types/styled-components/styled-components-tests.tsx

```diff
@@ -14,6 +14,7 @@ import styled, {
     withTheme,
     ThemeConsumer,
 } from 'styled-components';
+import * as SCMacro from 'styled-components/macro';
```

Comment:
I would suggest to extract tests of macro to a separate file. Then you would not have to have alias exports like this and use standard `styled` and others.

Reply:
`styled-components-macro-tests.tsx`?

Reply:
I wasn't sure if this was possible, but yeah, that's better.